neuro_san/client/agent_cli.py

```diff
@@ -26,12 +26,13 @@
 from grpc import RpcError
 from grpc import StatusCode
 
+from leaf_common.config.file_of_class import FileOfClass
```

Comment:
Use leaf-common version.
This is one of the few places that FileOfClass is used for file references not within the neuro-san distribution.